tests/excerptexport/models/test_export_model.py

```diff
@@ -11,7 +11,7 @@
 @pytest.fixture
 def export_stale_status(mocker, export):
     def timestamp_model_mixin_save_side_effect(self, *args, **kwargs):
-        super(TimeStampModelMixin, self).save(*args, **kwargs)
```

Comment:
Oops: Zero-argument `super()` cannot be used in functions defined outside the class body and causes

    SystemError: super(): __class__ cell not found

Fix proposed in #810